docs/tutorials/upgrade.md

```diff
@@ -96,8 +96,16 @@ application:
   - Configuration changes:
     - `streaming.forceTransmuxTS` has been renamed to `streaming.forceTransmux`
       (deprecated in v4.3.0)
-    - `manifest.dash.manifestPreprocessor` callback now receives a type of `shaka.externs.xml.Node` instead of `Element`.
-    - `manifest.mss.manifestPreprocessor` callback now receives a type of `shaka.externs.xml.Node` instead of `Element`.
+    - `manifest.dash.manifestPreprocessor` callback now receives a type of
+      `shaka.externs.xml.Node` which is a regular object instead of an `Element`.
+      Attributes and child strings are parsed as is without sanitisation.
+      It is worth noting that use of previous element APIs like `getAttribute`,
+      `textContent`, etc. would have returned html decoded strings.
```

Comment:
I don't find this sentence to be clear enough.  Also, am I mistaken in thinking that textContent automatically decodes the contents?  Isn't it only attributes that have to be decoded by the caller?  Or are you referring to Node.textContent, which differs from TXml.getTextContents()?

How about:

> `getAttribute()` results must now be decoded if they might contain escape sequences.  You can use `shaka.util.StringUtils.htmlUnescape` for this purpose. 

Or, depending on what is the correct understanding of `textContent`, maybe:

> `getAttribute()` and `textContent` results must now be decoded if they might contain escape sequences.  You can use `shaka.util.StringUtils.htmlUnescape` for this purpose.

Reply:
That sounds good to me!